### DISCUSSION TITLE
GitHub Developer Pack Email Address Conflict

### DISCUSSION THREAD

**Discussion by Mohiit70** [community/community#62614] [GitHub Education]
### Select Topic Area

Question

### Body

Hey GitHub community,

I hope you are all doing well. I am facing an issue related to the GitHub Developer Pack and need some assistance.

I have two GitHub accounts, "MohitBisht700" and "Mohiit70". My main account is "Mohiit70", and I would like to avail the GitHub Developer Pack benefits with this account.

The problem is that I tried adding my college email address to my main account "Mohiit70," but I received an error stating that the email is already in use. It seems that the college email is currently connected to my other account "MohitBisht700," which I don't intend to use for the GitHub Developer Pack.

Could someone please guide me on how to resolve this email address conflict and use the GitHub Developer Pack benefits with my primary account "Mohiit70"? Is there a way to disconnect the college email from "MohitBisht700," or can I use a different email address (not connected to any GitHub account) to avail of the benefits?

I appreciate any help or insights you can provide to help me streamline my access to the GitHub Developer Pack resources.

Thank you all for your time and support.

Best regards,
Mohit Bisht

**Comment by shivvamm** [comment]
That's simple if you want to use your college email address for your main account just go to your secondary account which is i think MohitBisht700 then open the settings now go to  the Emails section on the left you'll see the email you can add any other email if there's only the  college email address and then remove college email  by deleting it now you can add it to the main account.
I think that will work for you :)

**Comment by PriyanshNegi** [comment]
Hello @Mohiit70 ,

You can follow below steps :

1. Log in to "MohitBisht700" acc -> go to your account settings -> update the email address associated with that account to a different one that you don't intend to use for availing the GitHub Developer Pack benefits.

2. Once the college email is disassociated from "MohitBisht700," you should be now able to add it to your primary account "Mohiit70" without any issues. Go to the settings of the "Mohiit70" account -> add the college email address there.

**Reply by Mohiit70** [reply]
its showing mail id has already been used to obtain an academic discount on GitHub account(s) MohitBisht700.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Mohiit70, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by Mohiit70** [reply]
I apologize for not asking my question in applicable category. Can you please help me with resolving this email issue.

Thank you for your understanding and support.
